### DISCUSSION TITLE
About Admin page

### DISCUSSION THREAD

**Discussion by sariaRiski02** [AksaraDiskret/AksaraDiskret#15] [Announcements]
maybe I'll create a new branch for this feature we're going to make.

**Comment by sariaRiski02** [comment]
do I have to wait for you to finish your task? then it was my turn. and try to explain about the "forget password" issue that you made.

**Comment by sariaRiski02** [comment]
i mean,..
cause our task connected. you make "feature add book" and me "edit book", so i have to wait to complete your task. correct?
mybe i will create my task if you tell me how structure table and what method do you use to save data pdf.

i hope you understand it. 
Chuakkkksss........ wkwkwk

**Reply by rezkywm** [reply]
It should be, if the project document is complete. But this is a super fast project to working on. No time for it wkwkwk.

**Comment by sariaRiski02** [comment]
You can check the book edit feature. I have checked this but only 1 time. I didn't really check it properly

**Reply by rezkywm** [reply]
SFSG.
